pkg/kubelet/pleg/generic.go

```diff
@@ -406,9 +406,9 @@ func (g *GenericPLEG) updateCache(pod *kubecontainer.Pod, pid types.UID) error {
 	// all containers again.
 	status, err := g.runtime.GetPodStatus(pod.ID, pod.Name, pod.Namespace)
 	if klog.V(6).Enabled() {
```

Comment:
Does it make sense to check `Enabled()` here? Since it will be producing an Error message

Reply:
It makes sense here because the parameters are different depending on the verbosity level.

I opened https://github.com/go-logr/logr/issues/126 to discuss whether this should be possible without the if check. But for now this if check is the only way to do it.